share/spice/octopart/octopart.js

```diff
@@ -43,16 +43,20 @@
                 sourceName: 'Octopart',
                 sourceUrl : 'http://octopart.com/search?q=' + api_result.request.q
             },
+         
             normalize: function(item) {
-
-                item = item.item; // our item object is wrapped in an "item" property
-
+               
+                 var item = item.item;
+                 
+                 if (!item.avg_price_v2[0]){
+                     return null;
+                 }
                 var images = get_images(item.imagesets),
                     datasheet = item.datasheets && item.datasheets[0];
 
                 return {
                     brand: item.brand.name,
-                    price: item.avg_price_v2[1] + ' $' + item.avg_price_v2[0].toFixed(2),
```

Comment:
This is actually still required. If you take a look at the API result for some other queries, like https://duckduckgo.com/js/spice/octopart/atmega you'll see that the value is usually several decimal places, so we need this to ensure it's rounded properly.